src/Container.php

```diff
@@ -21,6 +21,11 @@ class Container implements ContainerInterface, ArrayAccess
     protected $items = [];
 
     /**
+     * @var array - \League\Container\ServiceProvider[]
```

Comment:
Nothing, just a description pointer to what the array should contain since we don't have arrayof